recipes/xo/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.1.11" %}
+
+package:
+  name: xo
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/scopatz/xo.git
+  git_tag: 8f486a2709d376d00f98c09a86179bce2a99099a
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  run:
+    - python >=3.4
+    - urwid >=1.1.1
+    - pygments >=1.6
+  build:
```

Comment:
Apparently the example did this too, but would you mind swapping the order of build and run?